source/_integrations/tuya.markdown

```diff
@@ -28,6 +28,20 @@ The Tuya integration allows integrating all Powered by Tuya devices you have add
 
 {% include integrations/config_flow.md %}
 
+Consult the following table and enter your Tuya credentials.
+
+| Field | Description |
+| ------- | -------- |
+| Access ID and Access Secret| Go to your cloud project on [Tuya IoT Platform](https://iot.tuya.com/). Find the **Access ID** and **Access Secret** on the **Project Overview** tab.|
+| Mobile App | Must select the one you used to link devices on the Tuya IoT Platform. |
+| Country Code | The country you select on logging in to Tuya Smart or Smart Life app.|
+| Account | Tuya Smart or Smart Life app account. |
+| Password | The password of your app account. |
+
+<p class='note'>
+The app mentioned in the table must be the one you used to link devices on the Tuya IoT Platform.
```

Comment:
Feel weird that this needs to be a separate note? Why not tell that directly when writing the description of the mobile app field?